data/json/items/tool_armor.json

```
@@ -392,6 +392,285 @@
+    "color": "light_gray",
+    "covers": [ "eyes" ],
+    "coverage": 95,
+    "encumbrance": 6,
+    "warmth": 6,
+    "ammo": "battery",
+    "charges_per_use": 1,
+    "use_action": [ "CAMERA", "MP3" ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "holster": true,
+        "rigid": true,
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "item_restriction": [ "light_battery_cell", "light_atomic_battery_cell", "light_disposable_cell" ]
+      }
+    ],
+    "material_thickness": 1,
+    "environmental_protection": 1,
+    "flags": [ "ONLY_ONE", "ZOOM", "WATCH", "ALARMCLOCK", "OUTER", "FRAGILE", "SUN_GLASSES", "THERMOMETER", "HYGROMETER" ]
+  },
+  {
+    "id": "ar_glasses_t2_on",
+    "copy-from": "ar_glasses_t2",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "pair of camping AR glasses (on)", "str_pl": "pairs of camping AR glasses (on)" },
+    "description": "This pair of camping AR glasses is turned on and playing some great tunes, raising your morale steadily while on your person.  It runs through batteries quickly; you can turn it off by using it.  It also obscures your hearing.",
+    "power_draw": 1000,
+    "revert_to": "ar_glasses_t2",
+    "use_action": [ "MP3_ON" ],
+    "flags": [
+      "ONLY_ONE",
+      "ZOOM",
+      "WATCH",
+      "ALARMCLOCK",
+      "OUTER",
+      "FRAGILE",
+      "SUN_GLASSES",
+      "THERMOMETER",
+      "HYGROMETER",
+      "TRADER_AVOID"
+    ]
+  },
+  {
+    "id": "ar_glasses_t2_mil",
+    "type": "TOOL_ARMOR",
+    "category": "clothing",
+    "name": { "str": "military AR headset", "str_pl": "military AR headsets" },
+    "description": "A pair of military grade augmented reality headset.  It comes with an integrated night vision system that can't compare to a proper NVG, a two-way radio and a few gadgets.",
+    "weight": "500 g",
+    "volume": "500 ml",
+    "price": 90000,
+    "price_postapoc": 500,
+    "to_hit": -2,
+    "material": [ "steel", "kevlar", "plastic" ],
+    "symbol": "[",
+    "looks_like": "glasses_safety",
+    "color": "light_gray",
+    "armor_portion_data": [
+      { "covers": [ "eyes" ], "coverage": 95, "encumbrance": 6 },
+      { "covers": [ "head" ], "coverage": 15, "encumbrance": 2 }
+    ],
+    "warmth": 8,
+    "ammo": "battery",
+    "charges_per_use": 1,
+    "use_action": [
+      {
+        "type": "transform",
+        "msg": "You activate your %s night vision system.",
+        "target": "ar_glasses_t2_mil_on",
+        "menu_text": "Turn NV on",
+        "active": true,
+        "need_charges": 1,
+        "need_charges_msg": "The %s's batteries are dead."
+      },
+      "CAMERA",
+      "RADIO_OFF"
+    ],
+    "material_thickness": 1,
+    "environmental_protection": 1,
+    "flags": [
+      "USE_UPS",
+      "NO_UNLOAD",
+      "NO_RELOAD",
+      "ONLY_ONE",
+      "TWO_WAY_RADIO",
+      "ZOOM",
+      "SUN_GLASSES",
+      "RAINPROOF",
```

```suggestion
      "WATERPROOF",
```
`RAINPROOF` flag is making you wholly immune to rain, regardless of which bodypart is covered. It doesn't make sense for glasses to make you immune to rain.